cargo-shuttle/src/init.rs

```diff
@@ -248,12 +188,12 @@ fn copy_template(src: &Path, dest: &Path) -> Result<()> {
         let entry_dest = dest.join(&entry_name);
 
         if entry_type.is_dir() {
-            if entry_name == ".git" {
+            if git_policy == GitDir::Ignore && entry_name == ".git" {
                 continue;
             }
 
             // Recursion!
-            copy_template(&entry.path(), &entry_dest)?;
+            copy_dirs(&entry.path(), &entry_dest, git_policy)?;
```

Comment:
Recursive copy could go infinite with a symlink? :thinking:

Reply:
Yes, I think `cargo-generate` filtered out symlinks beforehand. I'll add a check.

Reply:
`is_dir` and `is_file` don't return true for symlinks (as described [here](https://doc.rust-lang.org/std/fs/struct.FileType.html#method.is_symlink)). A warning would be nice though.